custom/aaa/static/aaa/spec/models/child.spec.js

```diff
@@ -55,7 +55,7 @@ describe('Child models', function () {
             );
             assert.equal('1', childListModel.id);
             assert.equal('<a href="unified_beneficiary_details/child/1/?month=3&year=2019">test Name</a>', childListModel.name());
-            assert.equal('1 Yr 2 Mon', childListModel.age());
+            assert.equal('14 Yr', childListModel.age());
```

Comment:
According to https://drive.google.com/file/d/1iRlFQfi0Jxf6eTgDO45V9fcvvVCwy47f/view this should be displaying in years and months, so 1 yr 2 mon would be correct.

I think that this needs to be changed in the dbaccessor so that its passing back the number of months instead of the year